modules/distribution/src/repository/resources/dbscripts/apimgt/oracle_rac.sql

```diff
@@ -411,13 +413,13 @@ CREATE OR REPLACE TRIGGER SP_REQ_PATH_AUTH_TRIG
 ALTER TABLE SP_REQ_PATH_AUTHENTICATOR ADD CONSTRAINT REQ_AUTH_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE
 /
 CREATE TABLE SP_PROVISIONING_CONNECTOR (
-            ID INTEGER,
-            TENANT_ID INTEGER NOT NULL,
+	    	ID INTEGER,
+	    	TENANT_ID INTEGER NOT NULL,
             IDP_NAME VARCHAR (255) NOT NULL ,
-            CONNECTOR_NAME VARCHAR (255) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            IS_JIT_ENABLED CHAR(1) DEFAULT '0',
-        BLOCKING CHAR(1) DEFAULT '0',
+	    	CONNECTOR_NAME VARCHAR (255) NOT NULL ,
+	    	APP_ID INTEGER NOT NULL,
+	    	IS_JIT_ENABLED CHAR(1) DEFAULT '0',
+		BLOCKING CHAR(1) DEFAULT '0',
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04